langchain4j-mcp/src/test/java/dev/langchain4j/mcp/client/integration/McpHeadersStreamableHttpTransportIT.java

```diff
@@ -36,7 +47,16 @@ static void tearDown() throws Exception {
             mcpClient.close();
         }
         if (process != null) {
-            process.destroyForcibly();
+            destroyProcessTree(process);
```

Comment:
I don't think this is necessary - could you elaborate on the issue where it didn't kill jbang properly? The server is just a single process, no need to kill the whole tree. Furthermore, if we needed to apply this change, we would need to do the same in all tests, not just this one specific test (but I don't think it's necessary, the test suite has been around for a long time and nobody complained about this issue so far).

Reply:
Ah ok, good catch then! I didn't try on Windows, I use Linux and apparently the behavior is different.
In that case, could you please apply this change across all tests in the `dev.langchain4j.mcp.client.integration` package? Probably extract the method for destroying the process tree into a separate class that can be reused because otherwise it would be duplicated many times.